department-of-veterans-affairs/vets-website
# Misc Updates

## Summary
On the Pension Benefits application(21P-527EZ)
- Update the page order
- Update list loop view components
- Update page title casings
- Use object keys in radio schemas
- Fix dependent disabled logic

## How to run in local environment
1. Check out this branch locally
2. In your local environment, set the `pension_form_enabled` flipper to 'enabled' at http://localhost:3000/flipper/features/pension_form_enabled
3. Run `vets-website` and `vets-api`

## How to verify
1. Go to `http://localhost:3001/pension/application/527EZ` in your browser
2. Start or continue the application
3. Verify the order of the chapters and pages

## Related issue(s)
[Epic in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/63349)

## Testing done


## What areas of the site does it impact?
Pension Benefits Application

## Acceptance criteria
[Ticket in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/71345)

### Quality Assurance & Testing

### Error Handling


### Authentication
